README.md

```diff
@@ -20,6 +20,8 @@ trait Cache[F[_], K, V] {
 
   def get(key: K): F[Option[V]]
 
+  def getOrElse(key: K, default: => V): F[V]
```

Comment:
why not `def getOrElse(key: K, default: => F[V]): F[V]` ?

Reply:
Good question :) will do